schemas/create-aliquot-in-mlwh/1-create-aliquot-in-mlwh.avsc

```diff
@@ -26,7 +26,7 @@
             "doc": "The LIMS system that the aliquot was created in"
         },
         {
-            "name": "limsUuid",
+            "name": "aliquotUuid",
             "type": {
                 "type": "string",
                 "logicalType": "uuid"
```

Comment:
I thought we were changing this to `aliquotId` and it wasn't going to be a UUID any more?

Reply:
No, apparently it needs to be a UUID _and_ and ID. @seenanair might be able to give a better explanation :)

Reply:
OK, I thought Traction didn't have a UUID for the aliquot.  Does that mean we should have a second field as well then?

Reply:
ID is not added now. It only includes UUID.
After discussion with Steve, this has been reverted back to UUID  because: 1) ID is specifically related to LIMS, and 2) IDs can change or disappear. Therefore, UUIDs seem as a better option as they are ubiquitous. 
Now, a UUID is generated for an aliquot when it is created, and a rake task has been added to fill in UUIDs for all existing aliquots. This allows us to use the initial schema definition with UUIDs.